samples/add-point-cloud-layer-from-file/src/main/java/com/esri/arcgismaps/sample/addpointcloudlayerfromfile/screens/AddPointCloudLayerFromFileScreen.kt

```diff
@@ -0,0 +1,60 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.addpointcloudlayerfromfile.screens
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.SceneView
+import com.esri.arcgismaps.sample.addpointcloudlayerfromfile.components.AddPointCloudLayerFromFileViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the Add point cloud layer from file sample.
+ */
+@Composable
+fun AddPointCloudLayerFromFileScreen(sampleName: String) {
+    val viewModel: AddPointCloudLayerFromFileViewModel = viewModel()
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = { paddingValues ->
+            Column(modifier = androidx.compose.ui.Modifier.padding(paddingValues)) {
```

Comment:
This could be simplified without Column? Also can avoid full package name `androidx.compose.ui.Modifier`